src/qml/QFieldCloudPopup.qml

```diff
@@ -438,7 +438,17 @@ Popup {
             }
             wrapMode: Text.WordWrap
             horizontalAlignment: Text.AlignHCenter
-            Layout.bottomMargin: 10
+            Layout.fillWidth: true
+          }
+
+          Text {
```

Comment:
I would move this on top of the screen, so people never miss it.

Reply:
I'm -0.5 on having it on top. I think this is meant to be a background task that's out of the way and that users shouldn't really have to think about or care for. 

I would prefer keeping it at the bottom of the screen for now and see how people react to that during the initial public beta phase.

Reply:
Can we change the icon of the cloud icon on top to something that indicates there is a background task. Then I am fine with the bottom position?